### PR TITLE
ID based search for JSTOR

### PR DESCRIPTION
**Pull request by joethei**
implemented in https://github.com/JabRef/jabref/pull/7145

### CODE REVIEW

File: en/collect/add-entry-using-an-id.md [hunk 1]
```diff
@@ -60,6 +60,12 @@ The [International Association for Cryptologic Research](https://www.iacr.org/)
 
 ID search is carried out using the Cryptology ePrint ID.
 
+### JSTOR
+
+Jstor is a digital library of academic content in many disciplines.
```
Comment (mlep) [inline en/collect/add-entry-using-an-id.md:65]: You could take the same description of Jstor as in https://docs.jabref.org/collect/import-using-online-bibliographic-database

File: en/collect/add-entry-using-an-id.md [hunk 2]
```diff
@@ -60,6 +60,12 @@ The [International Association for Cryptologic Research](https://www.iacr.org/)
...
+ID search is carried out using either the [stable url](https://support.jstor.org/hc/en-us/articles/115004872507-Citation-Management-Permanently-Linking-to-Content-on-JSTOR) or a DOI.
```
Comment (mlep) [inline en/collect/add-entry-using-an-id.md:66]: Please, clarify what is the "stable url" compared to DOI: [Jstor stable url].

File: en/collect/add-entry-using-an-id.md (hunk 1 above)
Comment (mlep) [reply en/collect/add-entry-using-an-id.md:65]: Please, put the 2 sentences in different paragraphs, so as to reproduce the layout used for the other search engines.

File: en/collect/add-entry-using-an-id.md (hunk 2 above)
Comment (joethei) [reply en/collect/add-entry-using-an-id.md:66]: The stable url is basically the same as a doi.

Some stable urls only contain the doi(http://www.jstor.org/stable/10.1086/501484 for example)
while most use a common prefix(https://www.jstor.org/stable/90002164  = **10.2307**/90002164)

Comment (mlep) [reply en/collect/add-entry-using-an-id.md:67]: In this help section, that would be too much information. I suggest : "ID search is carried out using either the [Jstor stable url](https://support.jstor.org/hc/en-us/articles/115004872507-Citation-Management-Permanently-Linking-to-Content-on-JSTOR) or the DOI."

Comment (joethei) [reply en/collect/add-entry-using-an-id.md:67]: Which is what if have written.

Comment (mlep) [reply en/collect/add-entry-using-an-id.md:67]: Almost :wink: : I recommended [stable **jstor** url]. Plus "the" or "a" for both.

Comment (joethei) [reply en/collect/add-entry-using-an-id.md]: oh, you are correct, fixed.

**Review by mlep — APPROVED** [review]
Thanks for your involvement in improving JabRef!

**Comment by koppor** [comment]
Need to postpone a merge, because we needed to disable JSTOR (at https://github.com/JabRef/jabref/pull/7229), because JabRef gets blocked by JSTOR. We currently have no clue how to fix it.

**Pull request closed without merging**